package.json

```
@@ -136,11 +136,13 @@
           "default": "dagre",
           "enum": [
             "dagre",
-            "elk"
+            "elk",
+            "tala"
           ],
           "enumDescriptions": [
             "The directed graph layout library Dagre",
-            "Eclipse Layout Kernel (ELK) with the Layered algorithm"
+            "Eclipse Layout Kernel (ELK) with the Layered algorithm",
+            "General Orthogonal Layout Engine - available if installed"
```

```suggestion
            "Terrastruct's AutoLayout Approach (TALA) - available if installed"
```
maybe description from something here: 
https://terrastruct.com/tala/